sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -349,7 +349,7 @@ namespace Azure { namespace Storage { namespace Blobs {
       bool operator!=(const DeleteSnapshotsOption& other) const { return !(*this == other); }
       const std::string& Get() const { return m_value; }
       const static DeleteSnapshotsOption IncludeSnapshots;
-      const static DeleteSnapshotsOption Only;
+      const static DeleteSnapshotsOption OnlySnapshots;
```

Comment:
This naming is inconsistent with other languages.  I like what you've chosen better than other languages, but maybe there was a reason for their names.  

@tg-msft any opinion on this?

Reply:
I made this change to align with .Net

Reply:
in .Net code:

```
#region enum DeleteSnapshotsOption
namespace Azure.Storage.Blobs.Models
{
    /// <summary>
    /// Required if the blob has associated snapshots. Specify one of the following two options: include: Delete the base blob and all of its snapshots. only: Delete only the blob's snapshots and not the blob itself
    /// </summary>
    public enum DeleteSnapshotsOption
    {
        /// <summary>
        /// none
        /// </summary>
        None,

        /// <summary>
        /// include
        /// </summary>
        IncludeSnapshots,

        /// <summary>
        /// only
        /// </summary>
        OnlySnapshots
    }
}
```